cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,16 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connections, OperationalError
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **kwargs):
+        while True:
+            try:
+                connections["default"].ensure_connection()
+                break
+            except OperationalError:
+                self.stdout.write("Server loading...")
+                time.sleep(1)
```

Comment:
It might be beneficial to add a maximum retry limit or a timeout to avoid an infinite loop in case the database is unavailable for an extended period.